packages/run-it/src/components/ShowResponse/responseUtils.tsx

```diff
@@ -117,7 +113,7 @@ const ShowImage = (response: IRawResponse) => {
 
 /** A handler for HTTP type responses */
 const ShowHTML = (response: IRawResponse) => (
-  <CodeStructure language={'html'} code={response.body.toString()} />
+  <CodeDisplay language={'html'} code={response.body.toString()} />
```

Comment:
For some reason I seem to have broken `ShowResponse.spec.tsx` test with this change... I spent a while looking into it but could not figure out why...

Reply:
BTW, I like the name `CodeDisplay`